src/js/components/common/dialogs/deviceconnectiondialog.js

```diff
@@ -174,8 +175,9 @@ export const DeviceConnectionDialog = ({ onCancel }) => {
                 Next
               </Button>
             ) : (
-              <Button variant="contained" disabled={!onboardingComplete} onClick={onCancel}>
+              <Button variant="contained" disabled={!onboardingComplete} onClick={onCancel} style={{ height: '37px' }}>
                 {onboardingComplete ? 'Close' : 'Waiting for device'}
+                <Loader show={!onboardingComplete} small style={{ marginTop: '-5px' }} />
```

Comment:
```suggestion
                <Loader show={!onboardingComplete} small table />
```
might work as well - my environment isn't fully back yet, so just a guess... (that would probably also get around the height adjustment)